problem_builder/mrq.py

```diff
@@ -121,17 +120,15 @@ def calculate_results(self, submissions):
 
         for choice in self.custom_choices:
             choice_completed = True
-            choice_tips_html = []
             choice_selected = choice.value in submissions
 
             if choice.value in self.required_choices:
                 if not choice_selected:
                     choice_completed = False
             elif choice_selected and choice.value not in self.ignored_choices:
                 choice_completed = False
-            for tip in self.get_tips():
-                if choice.value in tip.values:
-                    choice_tips_html.append(tip.render('mentoring_view').content)
```

Comment:
You have removed `tip.render('mentoring_view').content` and `loader.render_template('templates/html/tip_choice_group.html', ...)` from here, but you did not add that back in anywhere else. That's probably why so many integration tests etc. are failing. You must preserve the existing functionality in the student_view and the corresponding HTML (i.e. when you test this XBlock in the LMS/Studio, tips should still be working just as they always are).

When using this XBlock via it's student_view, at some point the tips must be converted to HTML using those templates. You can do that here (as was the case), elsewhere in python, or even in the overall HTML template, but it must be done somewhere.

It's only student_view_data where we definitely want none of this HTML wrapping the tips.

Reply:
@bradenmacdonald Ya I realised this over the weekend. I misunderstood your [reply](https://tasks.opencraft.com/browse/OC-2958?focusedCommentId=45063&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-45063) and assumed that the HTML wasn't important anywhere.